src/test/java/com/rakovets/course/java/core/practice/oop/principles/PersianTest.java

```diff
@@ -0,0 +1,61 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.stream.Stream;
+
+public class PersianTest {
+
+    @Test
+    void mew() {
```

Comment:
"Violation: 'nameMethodTest' was not used for test-method naming
Solution: Rename method
"